hack/run-functests.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 GINKGO_SUITS=${GINKGO_SUITS:-functests}
-GINKGO_SUITS="functests/0_config functests/2_performance_update"
 LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
```

Comment:
can we please move this change in the parent patch? even though the end result is the same, it is better to avoid introducing this change in the first place.

Reply:
done